src/_includes/core-nodes/split-use-case.md

```diff
@@ -97,8 +89,6 @@ the catch-all category. Node-RED uses hashes, key-value pairs of property names
 and values. JSON is example of this. In the example below, we'll split the mapping
 of the words "one" and "two" versus the number values.
 
-![Split an object](./images/split-node-hash.png "Split arbitrary objects")
-
-```json
+{% renderFlow 500 %}
```

Comment:
As the flow was exported from a flow that had many other nodes etc present, the coordinates start at Y=340. That means that the view needs to be at least 400 px tall to show the flow. It makes the top part awkward.

cc @joepavitt @Steve-Mcl

Reply:
Already done in the description :)